edge-modules/functions/binding/src/Microsoft.Azure.WebJobs.Extensions.EdgeHub/Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj

```diff
@@ -18,14 +18,20 @@
     <Title>Microsoft Azure Edge Function extension</Title>
     <Authors>Microsoft</Authors>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
-    <PackageLicenseUrl>https://github.com/Azure/iotedge/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
     <Description>This package contains binding extensions for IoTEdge.</Description>
     <PackageIconUrl>http://go.microsoft.com/fwlink/?LinkID=288890</PackageIconUrl>
+    <PackageIcon>icon.png</PackageIcon>
     <PackageProjectUrl>https://github.com/Azure/iotedge</PackageProjectUrl>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <PackageTags>IoT Microsoft Azure IoTEdge Functions</PackageTags>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Include="..\..\..\..\..\LICENSE" Pack="true" PackagePath=""/>
+    <None Include="images\icon.png" Pack="true" PackagePath=""/>
```

Comment:
What's the image for and where will it be displayed?

Reply:
I'm pretty sure it's displayed in nuget.org search results, not sure if it's displayed anywhere else. I'm a little annoyed at having to check in a png file, especially in such a seemingly trivial case, but it preserves existing behavior and it's very minor, so the easiest path.